services/ux-backend/main.go

```diff
@@ -61,8 +62,15 @@ func main() {
 		klog.Info("shutting down!")
 		os.Exit(-1)
 	}
+	//TODO: deprecate in future release
 	http.HandleFunc("/onboarding-tokens", func(w http.ResponseWriter, r *http.Request) {
-		onboardingtokens.HandleMessage(w, r, config.tokenLifetimeInHours)
+		clientonboardingtokens.HandleMessage(w, r, config.tokenLifetimeInHours)
```

Comment:
This will result in wrong error handling because the code inside assumes a `client-onboarding-tokens` endpoint